evm/deploy-gateway-v5.0.x.js

```diff
@@ -29,7 +29,7 @@ function getProxyParams(governance, mintLimiter) {
 async function deploy(config, options) {
     const { chainName, privateKey, reuseProxy, verify, yes } = options;
 
-    const contractName = 'AxelarGateway';
+    const contractName = options.amplifier ? 'AxelarAmplifierGateway' : 'AxelarGateway';
```

Comment:
the gateway is the same, no need for a separate identifier i think

Reply:
aren't we going to have two separate gateways for chains that are migrating from consensus to amplifier?